File: fc-chat/demo/part3/ch3/05-1-message-system-add-channel/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/JoinResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.dto.domain.ChannelId;

public class JoinResponse extends BaseMessage {

  private final ChannelId channelId;
  private final String title;

  public JoinResponse(ChannelId channelId, String title) {
    super(MessageType.JOIN_RESPONSE);
    this.channelId = channelId;
    this.title = title;
  }

  public ChannelId getChannelId() {
    return channelId;
  }

  public String getTitle() {
    return title;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/JoinResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.constant.MessageType
import com.yscorp.withpush.messagesystem.dto.domain.ChannelId


class JoinResponse(val channelId: ChannelId, val title: String) : BaseMessage(MessageType.JOIN_RESPONSE) {
}
